src/main/features/response/form/models/defendantPaymentPlan.ts

```diff
@@ -0,0 +1,94 @@
+import { LocalDate } from 'forms/models/localDate'
+import { PaymentSchedule } from 'ccj/form/models/paymentSchedule'
+import { IsDefined, IsIn, IsPositive, ValidateNested, MaxLength } from 'class-validator'
+import { IsValidYearFormat } from 'forms/validation/validators/isValidYearFormat'
+import { IsValidLocalDate } from 'forms/validation/validators/isValidLocalDate'
+import { IsFutureDate } from 'app/forms/validation/validators/dateFutureConstraint'
+import { IsLessThanOrEqualToSumOf } from 'forms/validation/validators/isLessThanOrEqualToSumOf'
+import { Fractions } from 'forms/validation/validators/fractions'
+
+export class ValidationErrors {
```

Comment:
can you please extract the paymentPlan validation messages to somewhere common as they look the same to me?